tables-to-go.go

```diff
@@ -37,6 +37,7 @@ func NewCmdArgs() (args *CmdArgs) {
 	flag.StringVar(&args.Schema, "s", args.Schema, "schema name")
 	flag.StringVar(&args.Host, "h", args.Host, "host of database")
 	flag.StringVar(&args.Port, "port", args.Port, "port of database host, if not specified, it will be the default ports for the supported databases")
+	flag.StringVar(&args.SSLMode, "sslmode", args.SSLMode, "connect to database using secure connection. Supported values: 'require' (default), 'verify-full', 'verify-ca', and 'disable'")
```

Comment:
Let's not mention _all_ the supported options, since these are quite a few: https://www.postgresql.org/docs/current/libpq-ssl.html

We can mention that the value is passed as is to the underlying driver and hence supports all available options.

Reply:
What do you suggest? Should I just mention something like this for the flag's description:

Option: 1
```
connect to database using secure connection. The value will be passed as is to the underlying driver.
```

.. but the user might not know the supported values. Sometimes when I see this flag I always think the value should be (True / False) !!

Or

Option: 2
```
connect to database using secure connection. Please refer to this site for supported values: https://www.postgresql.org/docs/current/libpq-ssl.html
```

Reply:
Oh good point for confusion with booleans. What I usually do is to checked how the `--help` output will look like and experiment a bit with different format options. Based on that I think we could use the following:

```
"Connect to database using secure connection. (default \"disabled\")\nRefer to this site for supported values: https://www.postgresql.org/docs/current/libpq-ssl.html"
```

Note on the line-break between the sentences. This makes it look like the following, which I think looks good enough?

```
  -sslmode string
    	Connect to database using secure connection. (default "disabled")
    	Refer to this site for supported values: https://www.postgresql.org/docs/current/libpq-ssl.html (default "disabled")
```

We could even go a step further and add `The value will be passed as is to the underlying driver.` back, with newline...

```
"Connect to database using secure connection. (default \"disabled\")\nThe value will be passed as is to the underlying driver.\nRefer to this site for supported values: https://www.postgresql.org/docs/current/libpq-ssl.html"
```
... and following result:
```
  -sslmode string
    	Connect to database using secure connection. (default "disabled")
    	The value will be passed as is to the underlying driver.
    	Refer to this site for supported values: https://www.postgresql.org/docs/current/libpq-ssl.html
```

WDYT?

Reply:
I'll go with your last suggestion.